Release Notes/601.md

```diff
@@ -2,8 +2,18 @@
 
 ## New APIs
 
+- `IntegerLiteralExprSyntax` and `FloatLiteralExprSyntax` now have a computed `representedLiteralValue` property.
+  - Description: Allows retrieving the represented literal value when valid.
+  - Issue: https://github.com/apple/swift-syntax/issues/405
+  - Pull Request: https://github.com/apple/swift-syntax/pull/2605
+
 ## API Behavior Changes
 
+- Moved `Radix` and `IntegerLiteralExprSyntax.radix` from `SwiftRefactor` to `SwiftSyntax`.
```

Comment:
@ahoppen @bnbarham not sure if this is a change or a new api?

Reply:
I think *API-Incompatible Changes* would be the correct section because technically somebody could be importing just `SwiftRefactor` but not `SwiftSyntax`, in which case their code would break. But I think that’s very unlikely since you can’t really do anything without also importing `SwiftSyntax`